templates/pages/reset-password.jsx

```diff
@@ -87,8 +87,10 @@ var ResetPassword = React.createClass({
         queryObj = Url.parse(window.location.href, true).query;
         ga.event({category: 'Reset Password', action: 'Successfully request new password'});
         if ( queryObj.android === 'true' ) {
-          window.location = "webmaker://login?mode=sign-in";
-          return;
+          if (navigator.userAgent.toLowerCase().indexOf("android") > -1) {
```

Comment:
Which browsers don't have window.navigator.userAgent?

afaict, it's supported pretty much everywhere: https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent